### DISCUSSION TITLE
Trying to look through abstractions.xunit with VS 2019

### DISCUSSION THREAD

**Discussion by jchesshir** [xunit/xunit#2416] [Question] [answered]
I just tried opening the solution file xunit.abstractions.sln.  The solution itself opened up all right, but the only project it references will not open.  It automatically produced an html file, UpgradeLog.htm, which indicates I need to run `dotnet migrate`, but the MS documentation for that command indicates that it was deprecated by .NET Core 3.0.  We are running .NET Core 5.0.

Any ideas on a better way to try to look through the xunit.abstractions code, or should I file a bug in the xunit/xunit project about it?

**Comment by bradwilson** [comment, marked as answer]
I have updated the project file appropriately. Thanks for bringing this to my attention!

https://github.com/xunit/abstractions.xunit/commit/335cd98c6b63c1402d7116e589502d3e48a23b5d
